assets/tests/components/swingsView.test.tsx

```diff
@@ -221,7 +221,16 @@ describe("SwingsView", () => {
     expect(tree).toMatchSnapshot()
   })
 
-  test("opens VPP when clicking an active run", () => {
+  test("opens VPP when clicking an active run and sends Fullstory event", () => {
+    const originalFS = window.FS
+    const originalUsername = window.username
+    window.FS = { event: jest.fn(), identify: jest.fn() }
```

Comment:
this may not be the time to do it, but we have enough Fullstory custom events in the frontend that we would benefit by writing helpers for them